src/cli-validator/utils/duplicateProperties.js

```diff
@@ -0,0 +1,123 @@
+module.exports = function(input, chalk, validFile) {
+  const stringObj = JSON.stringify(input);
+  const splits = stringObj.trim().split(/,|{|}/);
+  const arr = [];
+
+  // Parsing through the splits and store key and values
+  // that are present after we encounter "properties"
+  for (let i = 0; i < splits.length; i++) {
+    if (splits[i].includes('properties')) {
+      let tempKey = splits[i];
+      tempKey = tempKey.replace(':', '');
+      tempKey = tempKey.replace(/"|'/g, '');
+      tempKey = tempKey.trim();
```

Comment:
I feel like this stuff is unnecessary - we should be able to parse through a JSON object rather than a string